HIP/hip-1068.md

```diff
@@ -0,0 +1,528 @@
+    1. The transaction succeeds if the allowance is present and has either:
+        1. unlimited remaining allowance; or
+        2. A remaining balance greater than the current transaction fee
+    2. Otherwise the transaction fails.
+5. Where the transaction succeeds, the HBAR fees for the transaction are deducted from the Fee Sponsors account (i.e.
+    the software provider).
+    1. The customer’s HBAR account balance (used to sign the transaction) is left untouched
+    2. The customer’s HBAR balance can be zero.
+6. The transaction is successful.
+
+### Protobuf
+
+A new new allowance scope will be added to help define which type of fee an allowance can be used for
+```protobuf
+    enum AllowanceScope {
+        /**
+         * Identifies the allowance value may be used to satisfy crypto transfer transactions.
+         */
+        CRYPTO_TRANSFER = 0;
+
+        /**
+         * Identifies the allowance value may be used to satisfy HAPI transactions fees due by the node, network and
+         * service.
+         */
+        TRANSACTION_FEE = 1;
+
+        /**
+         * Identifies the allowance value may be used to satisfy token custom fee due to the fee collector.
+         */
+        TOKEN_CUSTOM_FEE = 2;
+    }
+```
+
+Existing allowance types will need to be updated to specify the scopes
+```protobuf
+    /**
+    * An approved allowance of hbar transfers for a spender.
+    */
+    message CryptoAllowance {
+        ...
+
+        /**
+        * The applicable scope for this CryptoAllowance. 
+        */
+        repeated AllowanceScope scope = 3;
+    }
+```
+By default existing crypto allowances will map to a `CryptoAllowance` with scope `CRYPTO_TRANSFER`.
+
+```protobuf
+    /**
+    * An approved allowance of non-fungible token transfers for a spender.
+    */
+    message NftAllowance {
+        ...
+
+        /**
+        * The applicable scope for this NftAllowance. 
+        */
+        repeated AllowanceScope scope = 7;
+    }
+```
+By default existing nft allowances will map to an `NftAllowance` with scope `CRYPTO_TRANSFER`
+
+```protobuf
+    /**
+    * An approved allowance of fungible token transfers for a spender.
+    */
+    message TokenAllowance {
+        ...
+
+        /**
+        * The applicable scope for this TokenAllowance. 
+        */
+        repeated AllowanceScope scope = 5;
+    }
+```
+
+By default existing nft allowances will map to an `NftAllowance` with scope `CRYPTO_TRANSFER`
+
+> Note: After this HIP allowances will require explicit specification of the intended scope by the owner to set non
+`CRYPTO_TRANSFER` allowances.
+
+The updated TransactionBody will contain the sponsoring account identifier
+```protobuf
+    /**
+     * A value transfer sponsor claim.
+     */
+    message TransactionSponsorClaim {
+        /**
+         * The account a spender is noting has provided prior willingness through an transaction fee allowance to
```

Comment:
Typo: an -> a

I think the field description is a bit too complex and can be simplified. Part of the description can be moved in the description for the TransactionSponsorClaim operation